anchor/programs/concert-x/src/lib.rs

```diff
@@ -146,7 +147,10 @@ pub struct Concert {
     /// Unix timestamps when the campaign ends
     pub end_date: i64,               
     /// 0 = active, 1 = completed, 2 = cancelled
-    pub status: u8,                  
+    pub status: u8,
+    /// Concert contributors
+    #[max_len(100)]
+    pub contributors: Vec<Pubkey>,
```

Comment:
Does it make sense to make this a vec of a map with {Pubkey: contribution amount} key:value pairs? Since the logic currently allows people to donate any amount above the ticket price. We can also change the logic easily so that they can only contribute the price of the ticket and nothing above it. What do you think?